doc/classes/CodeEdit.xml

```diff
@@ -527,16 +527,16 @@
 	</methods>
 	<members>
 		<member name="auto_brace_completion_enabled" type="bool" setter="set_auto_brace_completion_enabled" getter="is_auto_brace_completion_enabled" default="false">
-			Sets whether brace pairs should be autocompleted.
+			If [code]true[/code], uses [member auto_brace_completion_pairs] to automatically insert the closing brace when the opening brace is inserted by typing or autocompletion. Also automatically removes the closing brace when using backspace on the opening brace.
```

Comment:
Looks good, but maybe add an explicit mention of quotes as well? As the setting also applies to closing quotes but this is not apparent from the name or description, making it hard to find if you're looking to stop the editor from inserting matching quotes. For example change to _"... insert the closing brace or quote when an opening one is inserted by... "_

Reply:
Check the Editor Settings description, this is the CodeEdit one. This will use whatever is set in the `auto_brace_completion_pairs`, so there may not be any quotations.

The Editor Settings one mentions quotations and comments.

https://github.com/godotengine/godot/pull/97656/files#diff-52f0c165415d19abd2d5586c8a168169a8d39716be86688a9117125409b421f0

Reply:
Right, I wasn't aware of the auto_brace_completion_pairs setting. Then I agree that your original wording is a better choice, even if this design choice with a configurable auto_brace_completion_pairs set and the naming of the settings makes it hard to find and understand these settings if you're looking to control the behavior regarding quotes. But this might then be an issue beyond this PR, and it's still a big improvement over the previous state, thanks!